.gitignore

```diff
@@ -1 +1,4 @@
 .DS_Store
+
+# congfig
+/.idea
```

Comment:
Please set your IDE to add a newline at the end of the file

Reply:
Solved